aip-console-tools-cli/src/main/java/com/castsoftware/aip/console/tools/commands/OnboardApplicationCommand.java

```diff
@@ -0,0 +1,121 @@
+package com.castsoftware.aip.console.tools.commands;
+
+import com.castsoftware.aip.console.tools.core.dto.DeepAnalyzeProperties;
+import com.castsoftware.aip.console.tools.core.dto.ExclusionRuleType;
+import com.castsoftware.aip.console.tools.core.dto.FastScanProperties;
+import com.castsoftware.aip.console.tools.core.dto.ModuleGenerationType;
+import com.castsoftware.aip.console.tools.core.services.ApplicationService;
+import com.castsoftware.aip.console.tools.core.services.JobsService;
+import com.castsoftware.aip.console.tools.core.services.RestApiService;
+import com.castsoftware.aip.console.tools.core.services.UploadService;
+import com.castsoftware.aip.console.tools.core.utils.Constants;
+import com.castsoftware.aip.console.tools.core.utils.VersionInformation;
+import com.castsoftware.aip.console.tools.providers.CliLogPollingProviderImpl;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import picocli.CommandLine;
+
+import java.io.File;
+
+@Component
+@CommandLine.Command(
+        name = "OnboardApplication",
+        mixinStandardHelpOptions = true,
+        aliases = {"Onboard-Application"},
+        description = "Creates an application with Fast-Scan to manage source code using a modern workflow. Performs the Deep-Analyze and then Publish data to CAST Imaging Console."
+)
+@Slf4j
+@Getter
+@Setter
+public class OnboardApplicationCommand extends BasicCollable {
+    @CommandLine.Option(names = {"-n", "--app-name"},
+            paramLabel = "APPLICATION_NAME",
+            description = "The Name of the application to analyze",
+            required = true)
+    private String applicationName;
+
+    @CommandLine.Option(names = {"-f", "--file"},
+            paramLabel = "FILE",
+            description = "A local zip or tar.gz file OR a path to a folder on the node where the source if saved")
```

Comment:
a) typo ? "if saved" -> "is saved"

b) "a path to a folder on the node where the source if (or "is" ?) saved" : does it mean that the path can not be elsewhere than on a node ? if no then this line should be reviewed

Reply:
ok